### ISSUE TITLE
Constructor for NtpClock::setup() missing

### ISSUE THREAD

**Issue by Heideachim** [bxparks/AceTime#26]
In v. 0.8.1 there is no constructor for ace_time::clock::NtpClock::setup(void). As mentioned in the documentation, this should be the constructor to use when there is already an existing WLAN connection.
There is only a constructor for NtpClock::setup(const char* ssid, const char* password).
Otherwise, this library is a wonderful thing! Thanks!

The code to add to NtpClock.h would be:
`
   void setup(void) {
      mUdp.begin(mLocalPort);

    #if ACE_TIME_NTP_CLOCK_DEBUG == 1
      #if defined(ESP8266)
        SERIAL_PORT_MONITOR.print(F("Local port: "));
        SERIAL_PORT_MONITOR.println(mUdp.localPort());
      #endif
    #endif

      mIsSetUp = true;
    }
`

**Comment by bxparks** [comment]
Hi, Is this problem on the master branch or the develop branch? I have not merged #24 into master, have not created a new release, because I'm on vacation with just a cell phone with only sporadic internet connection.

**Comment by bxparks** [comment]
I'm going to close this because this code is in the develop branch through #24. The new signature for `setup()` is:
```C++
    void setup(const char* ssid = nullptr, const char* password = nullptr,
        uint16_t connectTimeoutMillis = kConnectTimeoutMillis) {
```
which supports calling it with no arguments `setup()`.
This will be included in the next release, which I'll probably push out today.
